Author: Alexander Gerasiov <gq@debian,org>
	Florian Will <florian.will@gmail.com>
Bug-Debian: https://bugs.debian.org/786776
Last-Update: 2015-05-27
Subject: In the case db_dir is defined in config, but log_dir is not, upstream
 uses db_dir as log_dir. We think this is wrong for Debian version of minidlna. 
--- a/minidlna.c
+++ b/minidlna.c
@@ -757,10 +757,7 @@
 	}
 	if (log_path[0] == '\0')
 	{
-		if (db_path[0] == '\0')
 			strncpyt(log_path, DEFAULT_LOG_PATH, PATH_MAX);
-		else
-			strncpyt(log_path, db_path, PATH_MAX);
 	}
 	if (db_path[0] == '\0')
 		strncpyt(db_path, DEFAULT_DB_PATH, PATH_MAX);
